server/src/main/java/org/apache/lucene/index/ShuffleForcedMergePolicy.java

```diff
@@ -77,7 +77,7 @@ public CodecReader wrapForMerge(CodecReader reader) throws IOException {
 
                 @Override
                 public void setMergeInfo(SegmentCommitInfo info) {
-                    // Record that this merged segment is current as of this schemaGen:
+                    // record that this segment was merged with interleaved segments
```

Comment:
```suggestion
                    // Record that this segment was merged with interleaved segments.
```

In case we might want to diverge from upstream but improve on style ;]. Otherwise: Nevermind!

Reply:
Only if there is a *really* good reason. 

Style changes like this only cause unnecessary conflicts on subsequent patches, so I'd rather not change it.

Reply:
Sure. Thanks for the explanation!